http.go

```diff
@@ -44,13 +46,18 @@ type mqttStatus struct {
 	IsConnected bool `json:"is_connected"`
 }
 
+type updateHTTPPayload struct {
+	NewVersion string `json:"version"`
+}
+
 func newHealthResponse() *healthResponse {
 	data := new(healthzDataResponse)
 	data.Services = processManager.Services
 	data.Daemons = daemonStore
 
 	meta := new(healthzMetaResponse)
 	meta.MQTTStatus.IsConnected = processManager.MQTTClient.IsConnected()
+	meta.Version = version
```

Comment:
where is _version_ declared???

Reply:
@mtavano operator version, this is a meta